app/views/highlighting/styles.css.erb

```diff
@@ -9,6 +9,10 @@
 <%= resource_color_css("meeting_status", meeting_status) %>
 <% end %>
 
+<% Projects::Statuses::AVAILABLE.each do |project_status| %>
+<%= resource_color_css("project_status", project_status) %>
```

Comment:
Missing indentation
```suggestion
  <%= resource_color_css("project_status", project_status) %>
```

Reply:
@HDinger I'll keep this as-is, as I think want to avoid extra indentation in the generated CSS.